### DISCUSSION TITLE
Wix V5: <Files> with tree in subfolder, but don't want to include that subfolder

### DISCUSSION THREAD

**Discussion by fxtech** [wixtoolset/issues#8092] [Questions]
I'm new to v5, and having trouble with the \<Files\> element. I want to include everything in a subfolder 'dist' but the dist folder ends up in the TARGETDIR with everything in there. I don't want it to install the 'dist' folder - just the contents.
I've tried setting a bindpath (with -b dist) and then doing this:
`<Files Include="!(bindpath.dist)\**"/>`
But that doesn't work. Is there some other way to tell it to just include everything in the Include *folder* but not the folder itself?

**Comment by ds-andreasor** [comment]
Not sure about the workings of bindpath but a normal `<Files Include="C:\Build\Dist\**" />` works for me.

**Reply by fxtech** [reply]
When I use this:
` <Files Include="dist\**"/>`
Then my files end up in **C:\Program Files\Company\Product\dist**.
Perhaps it's because I'm using a relative path?

**Comment by barnson** [comment]
There are a couple of different code paths; one picked the wrong substring as the "root." I'm fixing that now.

**Reply by barnson** [reply]
https://github.com/wixtoolset/issues/issues/8096
